cms/templates/partials/home-page-catalog.html

```diff
@@ -0,0 +1,25 @@
+{% load wagtailimages_tags image_version_url wagtailcore_tags %}
+<div class="catalog-section">
+    <div class="container">
+        <div class="row">
+            {% for page_block in page.contents %}
+                {% with block=page_block.value %}
+                    <div class="col-sm-6 col-lg-4 p-0 d-flex flex-column">
+
+                        <img src="{% image_version_url block.page.thumbnail_image "fill-475x300" %}" alt="{{ block.page.title }}">
```

Comment:
`thumbnail_image` is not marked required so better to add condition else can raise exception.

Reply:
Updated with `mit-dome.png` as default